### ISSUE TITLE
DDA Compliance(Keyboard accessibility) issue for paper-password-input

### ISSUE THREAD

**Issue by NandiniK** [Collaborne/paper-password-input#20]
Hello ,
I am facing issue in keyboard accessibility testing for paper-icon-button. The label is not reading for paper-icon-button. Please help me with the fix.

**Comment by ankon** [comment]
Could you maybe show some code snippets and screenshots with what you're trying to do, and what you're expecting, and what actually happens?

**Comment by NandiniK** [comment]
I am using below code for password field with eye symbol.
```
<paper-password-input id="change_oldPwd" required="required" label="OldPassword" auto-validate></paper-password-input>
```
As per DDA requirement, on tab of eye icon, it has to read as show password/any text. Reading loud is not happening.

The fix may be - if u add alt property in paper-password-input and pass same alt property to paper-icon-button in paper-password-input component.

Please let me know if have to explain the issue in detail.

**Comment by ankon** [comment]
Couple of pointers for me:
1. We do not want to embed any text in the component, because that just gets us way to far into i18n/l10n issues. This means the information must come from the outside
2. We definitely need a alt/aria-* on the icon, and cannot simply hide if from a screenreader: The button is functional
3. There is a longer discussion in https://github.com/PolymerElements/iron-icon/issues/8 that seems relevant. The short version seems to be: "Add attributes for the alt text(s)"

I need to ponder a bit how to do that best, but a PR that would add an property for controlling the alt text would probably be a good start :)

**Comment by NandiniK** [comment]
Thanks for the quick response. Will agree on point 1. Will be waiting for this fix to come. Thank you

**Comment by NandiniK** [comment]
I was trying with title attribute. 

<paper-icon-button title="visibility" suffix icon="[[_getIcon(visible)]]" on-tap="_toggleVisibile" class="visibility-icon" **title="{{title}}"**></paper-icon-button> 

This will add tooltip and reads the text.
